pkg/gui/recent_repos_panel.go

```diff
@@ -1,24 +1,57 @@
 package gui
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
+	"sync"
 
 	"github.com/jesseduffield/generics/slices"
 	"github.com/jesseduffield/lazygit/pkg/commands"
 	"github.com/jesseduffield/lazygit/pkg/env"
+	"github.com/jesseduffield/lazygit/pkg/gui/presentation/icons"
 	"github.com/jesseduffield/lazygit/pkg/gui/style"
 	"github.com/jesseduffield/lazygit/pkg/gui/types"
 )
 
+func (gui *Gui) getCurrentBranch(path string) string {
+	if branch, err := gui.os.Cmd.New(
+		fmt.Sprintf("git -C %s rev-parse --abbrev-ref HEAD", gui.os.Quote(path)),
+	).RunWithOutput(); err == nil {
+		return strings.Trim(branch, "\n")
+	}
+	return ""
+}
+
 func (gui *Gui) handleCreateRecentReposMenu() error {
 	recentRepoPaths := gui.c.GetAppState().RecentRepos
 
+	currentBranches := sync.Map{}
+
+	wg := sync.WaitGroup{}
+	wg.Add(len(recentRepoPaths[1:]))
+
+	for _, path := range recentRepoPaths[1:] {
```

Comment:
could we assign `recentRepoPaths[1:]` to a new variable so that we don't need to do the `[1:]` more than once?